docs/jdbc.md

```diff
@@ -0,0 +1,495 @@
+# Connecting to SQL Server using JDBC
+
+## Setup and Usage
+
+### Adding the library as a dependency
+
+<!-- {x-release-please-start-version} -->
+
+### Maven
+Include the following in the project's `pom.xml`:
+
+##### Mysql
+```maven-pom
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>mysql-socket-factory-connector-j-8</artifactId>
+    <version>1.13.1</version>
+</dependency>
+```
+##### Maria DB
+
+```maven-pom
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>mariadb-socket-factory</artifactId>
+    <version>1.13.1</version>
+</dependency>
+```
+##### Postgres
+
+Include the following in the project's `pom.xml`:
+
+```maven-pom
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>postgres-socket-factory</artifactId>
+    <version>1.13.1</version>
+</dependency>
+```
+
+#### Gradle
+Include the following the project's `build.gradle`
+
+##### Mysql
+```gradle
+compile 'com.google.cloud.sql:mysql-socket-factory-connector-j-8:1.13.1'
+```
+
+##### Maria DB
+```gradle
+compile 'com.google.cloud.sql:mariadb-socket-factory:1.13.1'
+```
+*Note*: Also include the JDBC Driver for MariaDB, `org.mariadb.jdbc:mariadb-java-client:<LATEST-VERSION>`
+
+##### Postgres
+```gradle
+compile 'com.google.cloud.sql:postgres-socket-factory:1.13.1'
+```
+
+*Note*: Also include the JDBC Driver for PostgreSQL, `org.postgresql:postgresql:<LATEST-VERSION>`
```

Comment:
Please make the `Note` markdown consistent in this doc. Currently there are:

```
*Note*:
Note:
**Note**:
```

Reply:
Done, using `**Note:** `